Carlos/DiskCacheLevel.swift

```diff
@@ -161,7 +159,7 @@ public class DiskCacheLevel<K: StringConvertible, T: NSCoding>: CacheLevel {
     }
   }
   
-  private func setDataSync(data: T, key: K) {
+  private func setDataSync(data: T, key: K, promise: Promise<()>) {
```

Comment:
as a note for me, `setDataSync` can basically be inlined in `set` until `updateAccessDate` is added again